shared/src/chain_state.rs

```diff
@@ -354,11 +341,11 @@ impl ChainState {
 
     // Add a verified tx into pool
     // this method will handle fork related verifications to make sure we are safe during a fork
-    pub fn add_tx_to_pool(&self, tx: Transaction, cycles: Cycle) -> Result<Cycle, PoolError> {
+    pub fn add_tx_to_pool(&self, tx: Transaction, cycles: Cycle) -> Result<Cycle, Error> {
         let short_id = tx.proposal_short_id();
         let tx_size = tx.serialized_size();
         if self.reach_tx_pool_limit(tx_size, cycles) {
-            return Err(PoolError::LimitReached);
+            Err(InternalError::FullTransactionPool)?;
```

Comment:
TransactionPoolFull?

Reply:
I personally like `adj + noun` format... So most of the error names are in this format.

Reply:
ok